docs/content/data-capture.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Data capture"
+weight: 30
+---
+
+# Data capture
+
+Side-eye data capture operates by navigating the memory, starting from frames -
+i.e. currently executed functions, following nested structures and pointers.
```

Comment:
nit: how about `currently executing functions`? Something about `currently` with the past tense seems off.

Reply:
done